whole_body_controller/base:
  odom_x_joint: &odom_x_joint odom_x_joint
  odom_y_joint: &odom_y_joint odom_y_joint
  odom_z_joint: &odom_z_joint odom_z_joint
  joints:
    - *odom_x_joint
    - *odom_y_joint
    - *odom_z_joint
  odom_frame: odom
  base_frame: base_link
  K_x: {p: 2.2, d: 0.7}
  K_y: {p: 2.2, d: 0.7}
  K_z: {p: 2.2, d: 0.7}
  freq: 50.0
  T_delay: 0.0
  T_finish: 1.0
  S_x: 0.01
  S_y: 0.01
  S_z: 0.01